packages/amazonq/src/extension.ts

```diff
@@ -147,6 +147,9 @@ export async function activateAmazonQCommon(context: vscode.ExtensionContext, is
     // Hide the Amazon Q tree in toolkit explorer
     await setContext('aws.toolkit.amazonq.dismissed', true)
 
+    // set context var to check if its SageMaker AI or not
+    await setContext('aws.isSageMaker', isSageMaker())
```

Comment:
👍  This is done for Toolkit here: https://github.com/aws/aws-toolkit-vscode/blob/191c50488418c2ddecb830fbc26f2063257acc33/packages/core/src/extensionNode.ts#L124-L129

but currently `packages/amazonq/src/extension.ts` does not share a "common" startup logic w/ Toolkit, so we need this redundant code here.